datasets/us/ma/med_exclusions/us_ma_med_exclusions.yaml

```
@@ -0,0 +1,34 @@
+title: US Massachusetts Medicaid Terminated Provider List
+entry_point: crawler.py
+prefix: us-medma
+coverage:
+  frequency: daily
+  start: "2024-08-15"
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+summary: >
+  This dataset contains a list of providers who have been terminated for cause from Massachusetts's Medicaid program.
+description: |
+  The MassHealth program maintains a list of providers who have been suspended or excluded
+  from participating in the MassHealth program. This list is updated monthly and reflects
+  suspensions or exclusions effective on or after March 23, 2010.
```

When we're quoting verbatim it's best to use the quote >'s to make sure it's clear we're not trying to act like it's out wording.

```suggestion
  > The MassHealth program maintains a list of providers who have been suspended or excluded
  > from participating in the MassHealth program. This list is updated monthly and reflects
  > suspensions or exclusions effective on or after March 23, 2010. They are listed according to
  > the provider type at the time of such action.
```